src/rules/speech_control.ts

```diff
@@ -1178,4 +1178,36 @@ export function initRules_bc_speech_control() {
 			}, ModuleCategory.Rules);
 		}
 	});
+
+	registerRule("speech_garble_while_talking", {
+		name: "Force garbled speech",
+		type: RuleType.Speech,
+		loggable: false,
+		shortDescription: "force PLAYER_NAME to talk as if they were gagged",
+		longDescription: "This rule forces PLAYER_NAME to talk as if they were gagged, automatically garbling all of their speech. This rule does not affect OOC or, if \"Garble whispers while gagged\" is disabled, whispers.",
+		keywords: ["garble", "saying", "talking"],
```

Comment:
We suggest adding: `gagtalk`, `garbling`, `forced`
Also you don't need `garble`, because it already is substring of the rule's name